src/components/Conforma/ConformaViolationSummaryCard.tsx

```
@@ -0,0 +1,126 @@
+  CardFooter,
+  CardHeader,
+  CardTitle,
+  Content,
+  ContentVariants,
+  Skeleton,
+} from '@patternfly/react-core';
+import { ArrowRightIcon } from '@patternfly/react-icons/dist/esm/icons/arrow-right-icon';
+import { CheckCircleIcon } from '@patternfly/react-icons/dist/esm/icons/check-circle-icon';
+import { ExclamationCircleIcon } from '@patternfly/react-icons/dist/esm/icons/exclamation-circle-icon';
+import { t_global_icon_color_status_success_default as greenColor } from '@patternfly/react-tokens/dist/js/t_global_icon_color_status_success_default';
+import { t_global_icon_color_status_danger_default as dangerColor } from '@patternfly/react-tokens/dist/js/t_global_icon_color_status_danger_default';
```

ESLint

[error] `@patternfly/react-tokens/dist/js/t_global_icon_color_status_danger_default` import should occur before import of `@patternfly/react-tokens/dist/js/t_global_icon_color_status_success_default`

(import/order)